src/components/CodeHighlight/Languages/langMARC.js

```diff
@@ -0,0 +1,3 @@
+export const langMARC = code => {
```

Comment:
Add todo maybe that it needs to be changed later in order to really deal with MARC.

Reply:
Done